aom/src/main/java/com/nedap/archie/aom/CObject.java

```diff
@@ -277,14 +277,10 @@ public ConformanceCheckResult cConformsTo(CObject other, BiFunction<String, Stri
         if(!nodeIdConformsTo(other)) {
             return ConformanceCheckResult.fails(ErrorType.VSONI, I18n.t("Node id {0} does not conform to {1}", this.getNodeId(), other.getNodeId()));
         }
-        if(!occurrencesConformsTo(other)) {
-            return ConformanceCheckResult.fails(ErrorType.VSONCO, I18n.t("Occurrences {0} does not conform to {1}", this.getOccurrences(), other.getOccurrences()));
-        }
```

Comment:
@pieterbos Is this not going to give issues elsewhere when this check is removed here?

Reply:
Shouldn't. The problem is that if we do not remove it, `occurrences matches {0}` will no longer work if a mandatory parent node should be replaced with a new specialization. It is only possible to correctly check this if you have the sibling nodes in the flat archetype as well, or both the sibling nodes in the differential archetype plus the differential parent.

At first Vera fixed this by keeping it here, and adding the new check as an edge case when it failed here. However, that also caused some trouble, since in some situations a single replacement is valid, while the combination of all replacements is not at all, making this check meaningless.